modules/TaskListHeaderPresenter.js

```diff
@@ -0,0 +1,37 @@
+import iconArrowsCcw from '../src/icon-arrows-ccw.svg';
+
+class TaskListHeaderPresenter {
+  #completedCount;
+
+  #listStats;
+
+  #view;
+
+  constructor() {
+    this.#view = document.createElement('div');
+    this.#view.setAttribute('class', 'list-header');
+    const title = document.createElement('h2');
+    title.setAttribute('class', 'list-header');
+    title.innerText = 'Demo';
+    this.#view.appendChild(title);
+    this.#listStats = document.createElement('div');
+    this.#listStats.setAttribute('class', 'list-stats');
+    this.#view.appendChild(this.#listStats);
+    this.#completedCount = document.createElement('span');
+    this.#completedCount.setAttribute('class', 'changes-count');
+    this.#listStats.appendChild(this.#completedCount);
+    const icon = document.createElement('img');
+    icon.setAttribute('class', 'icon-arrows-ccw');
+    icon.setAttribute('src', iconArrowsCcw);
+    this.#listStats.appendChild(icon);
```

Comment:
Some of these variables are used only in one method, so please try to create these variables as local variables in the method that uses them.

Reply:
Thank you!